components/LiveTrack.js

```diff
@@ -40,7 +62,9 @@ export default ({ app_type_description = '', created_at, track } = {}) => !track
         </div>
         <div>
           {track.source.artist.name}
-
+        </div>
+        <div className="subtext">
+          {createVenueMarkup(track)}
```

Comment:
Instead of a function can we make this a component?

```
<VenueInfo track={track} />
```

and the function would look like

```
const VenueInfo = ({ track }) => {
  // fn contents
};
```